oak-blob-plugins/src/main/java/org/apache/jackrabbit/oak/plugins/blob/AbstractSharedCachingDataStore.java

```diff
@@ -24,10 +24,14 @@
 import java.io.IOException;
```

Comment:
Why is there a need to change here. A temp file is already being created here. It won't add any benefit to create a another temp file in the local file system anyways.